x/evm/ante/ctx.go

```diff
@@ -0,0 +1,20 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
```

Comment:
please leave an empty line between the license and the package name - this causes linter warnings for those of us who use actual IDEs 😇 

```suggestion
// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)

```